requirements-dev.txt

```diff
@@ -2,6 +2,7 @@
 
 # Note all of these are generated from poetry lock/projects via `make pkg-gen-requirements`
 -r requirements.txt
+-r tools/c7n_awscc/requirements.txt
```

Comment:
`tox -e docs` also expects this to be installed.   I wasn't sure if this was the require location to add it but it seemed reasonable

Reply:
This seems in line with the rest of the project references here, though I'm honestly not sure why we're pulling in these requirements files. The editable installs below should also bring in anything from those per-project generated requirements files :thinking:

Not a problem since it'll skip unnecessary reinstalls, just seemed odd at a glance :shrug: